### ISSUE TITLE
Performance issues while resizing browser

### ISSUE THREAD

**Issue by AlexKutkanych** [amcharts/amcharts4#3681]
**Bug description**

We have a LineChart on one of our pages. When you resize the browser window a long lag is happening (browser tab stops being interactive, vertical scrollbar disappears) and after 10-20 sec everything becomes fine (please see video below). 

We already tried Disabling autoresizing stated [here](https://www.amcharts.com/docs/v4/concepts/performance/) but no luck with that one.

Could you please help overcome this issue?

https://www.loom.com/share/37d9933020974f2086475995f83ee5fc

**Environment (if applicable)**

 - amcharts: v4.10.21
 - Chrome: v92.0.4515.131
 - React: v17.0.2

**Comment by martynasma** [comment]
I'm not sure how chart this simple can lag so much as in your video. Do you have a live demo for us to check?

**Comment by AlexKutkanych** [comment]
@martynasma we investigated deeply into this and it occured that it was mainly caused by a custom tooltip:

`series.adapter.add('tooltipText', () => {
      let text = '[bold]{dateX}[/]\n';
      chart.current.series.each(function(item) {
        text +=
          '[' + item.stroke.hex + ']●[/] ' + item.name + ': {' + item.dataFields.valueY + '}\n';
      });
      return text;
    });`

**Comment by martynasma** [comment]
Cool. Were you able to figure out the issue for you, or still need help?

**Comment by AlexKutkanych** [comment]
@martynasma can we render all the tooltips needed once and just show them on hover?

**Comment by martynasma** [comment]
No, I don't think so.

Tough, I don't understand how tooltip can be an issue. If you don't hover on the chart, it's not shown and not updated.

**Comment by martynasma** [comment]
I don't think we have changed anything with this adapter, or any of adapters, for ages.

Are you able to pinpoint which version introduced this lag for you?
